src/packages/solid/components/Column/Column.tsx

```diff
@@ -9,14 +26,20 @@ export interface ColumnProps extends IntrinsicNodeProps {
   /** Item index at which scrolling begins */
   scrollIndex?: number;
   scrollType?: 'alwaysScroll' | 'neverScroll' | 'lazyScroll';
+  onSelectedChanged?: (
+    this: ElementNode,
+    elm: ElementNode,
+    active: ElementNode,
+    selectedIndex: number,
+    lastSelectedIndex: number
+  ) => void;
 }
 
 const Column: Component<ColumnProps> = (props: ColumnProps) => {
-
   return (
     <SolidColumn
       {...props}
-      style={styles.Container}
+      style={props.style ? [props.style, styles.Container] : styles.Container}
```

Comment:
does this need to be a ternary? im curious if/why its an issue if theres an undefined value in the array. or is it just preferred that we dont supply an array if we're only providing one style set? 

also, is there a limit to the amount of objects in the array? this is a cool feature and im just thinking about how we can use it in the future

Reply:
Good questions - We could make this just an array - I'll do that on the next release of solid to support it...

No limit. Styles are applied in order as well - so props.style would override Container

Reply:
fantastic 🚀